packages/app/src/components/views/publication/PublicationsView.tsx

```diff
@@ -174,6 +175,23 @@ export const PublicationsView: React.FC<PublicationsViewProps> = ({ updateChainI
 
   return (
     <Page showBadge>
+      <Helmet>
+        <title>Tabula</title>
+
+        <meta property="og:title" content="tabula" />
+
+        <meta property="og:title" content="Tabula" />
+        <meta property="og:site_name" content="Tabula" />
+        <meta
+          property="og:description"
+          content="Instant web3 publications for writers, DAOs, and any Ethereum-based account."
+        />
+        <meta
+          name="description"
+          content="Instant web3 publications for writers, DAOs, and any Ethereum-based account."
+        />
+        <meta property="og:url" content="https://tabula.gg" />
+      </Helmet>
```

Comment:
Why do we need this here? This seems to just be resetting what we already have in `packages/app/public/index.html`.